tsconfig.json

```diff
@@ -1,28 +1,37 @@
 {
-	"compileOnSave": true,
+	// To update the compilation target, install a different version of @tsconfig/node... and reference it here
+	// https://github.com/tsconfig/bases#node-14-tsconfigjson
+	"extends": "@tsconfig/node14/tsconfig.json",
```

Comment:
As mentioned in #4 I believe we should still support Node 12:
https://www.iobroker.net/#en/download still suggests the `https://deb.nodesource.com/setup_12.x` script.